### ISSUE TITLE
Add Church & Murray Business Site Selection to Refs

### ISSUE THREAD

**Issue by jGaboardi** [pysal/spopt#380]
xref https://github.com/pysal/spopt/pull/374#issuecomment-1589061609

See this entry in [`spaghetti`](https://github.com/pysal/spaghetti/blob/d2b37701c734bab4c0fc9f5745961859adda998b/docs/_static/references.bib#L497)

```
@book{Church2009,
  address = {Hoboken},
  author = {Church, Richard L. and Murray, Alan T.},
  publisher = {John Wiley {\&} Sons, Inc.},
  title = {{Business Site Selection, Locational Analysis, and GIS}},
  year = {2009}
}
```

**Comment by jGaboardi** [comment]
@rongboxu This entry should go into `spopt/docs/_static/references.bib`

**Comment by jGaboardi** [comment]
Resolved via #382
